resource random_password not working in vm module

### Module Name

vm

### Module Version

0.11.0

### Terraform Version

latest

### Description

Get error on terraform init when using vm module


```
Run terraform validate
╷
│ Error: Unsupported argument
│ 
│   on .terraform/modules/common-rg.vm/main.tf line 43, in resource "random_password" "this":
│   43:   numeric     = true
│ 
│ An argument named "numeric" is not expected here.
╵
Error: Process completed with exit code 1.
```


### Example Terraform Configuration

```terraform

```

### Steps to Reproduce

_No response_

--- a/versions.tf
+++ b/versions.tf
@@ -9,7 +9,7 @@ terraform {
 
     random = {
       source  = "hashicorp/random"
-      version = ">= 2.2.0"
+      version = ">= 3.3.0"
     }
   }
 }

